### PR TITLE
Improve textual content

### PR DESCRIPTION
**Pull request by dodolab**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-| domain expertise   |   ★☆☆☆   |   ★★☆☆   |   ★★☆☆   |   ★★★☆   |   ★★★★   | Knowledge of your discipline Knowledge of your discipline (e.g. web development, microservices, databases), Language, Tools, Business and product sense |
+| domain expertise   |   ★☆☆☆   |   ★★☆☆   |   ★★☆☆   |   ★★★☆   |   ★★★★   | Knowledge of your discipline (e.g. web development, microservices, databases), Language, Tools, Business and product sense |
 | problem solving     |   ★☆☆☆   |   ★☆☆☆   |   ★★☆☆   |   ★★★☆  |   ★★★☆   | Architecture & design patterns, Breaking down large problems, Critical thinking, Analytical skills, Creativity, Decision making |
 | communication  |   ★☆☆☆   |    ★☆☆☆    |  ★☆☆☆   |   ★★★☆   |  ★★★★   | Documentation, Collaboration, Relationship-building, Interviewing, Listening, Empathy |
-| leadership  |   ★☆☆☆   |    ★☆☆☆    |   ★☆☆☆   |   ★★☆☆   |   ★★★★   |Accountability, Responsibility, Mentorship, Making hard decisions, Setting an example |
+| leadership  |   ★☆☆☆   |    ★☆☆☆    |   ★☆☆☆   |   ★★☆☆   |   ★★★★   |Accountability, Responsibility, Mentorship, Leading by example |
```
Comment (vorcigernix) [inline README.md:15]: What's the issue with making hard decisions?

File: badges/badges.md [hunk 2]
```diff
@@ -1,26 +1,25 @@
-# Badges!
+# Badges
```
Comment (vorcigernix) [inline badges/badges.md]: for this whole file I'd say that original sentences works better than your changes.

File: developers/level3.md [hunk 3]
```diff
@@ -43,7 +42,7 @@
 
 ### Red flags
 * Fails to identify or communicate big roadblocks
-* Us-vs-them attitude 
+* Us-VS-them attitude
```
Comment (vorcigernix) [inline developers/level3.md]: no, this is clearly wrong

File: developers/level3.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # Engineer L3
-> Level 3 is the most needed team member. It does not mean we will limit the growth to higher levels, but on this level, you actually contribute in main areas of productivity for a developer. 
+> Level 3 is the most needed team member. It does not mean we will limit the growth to higher levels, but on this level, you actually contribute in main areas of productivity as a developer.
```
Comment (vorcigernix) [inline developers/level3.md:2]: this is questionable, I'd say the original sentence is more clear in intent

File: developers/level2.md [hunk 5]
```diff
@@ -10,8 +10,8 @@
 
 ### Is expected to
 
-* Lean into challenges when they offer the potential to create large impact
-* Help grow the engineering team through involvement in the technology community (conferences, Socialbakers, blog posts, open source projects, etc) and referrals
+* Dive into challenges when they offer the potential to create a large impact
```
Comment (vorcigernix) [inline developers/level2.md]: I'd still rather use Lean

File: badges/badges.md (hunk 2 above)
Comment (vorcigernix) [reply badges/badges.md]: could you send additional pull request for that? I'd like to involve Gil into review

**Review by vorcigernix — CHANGES_REQUESTED** [review]
Great work, some nitpicks, please read the comments and let's continue the discussion on Slack

File: README.md (hunk 1 above)
Comment (dodolab) [reply README.md:15]: It's not clear what does it mean and somebody may take it wrong. I thought that `Leading by example` would do it more justice

File: developers/level3.md (hunk 3 above)
Comment (dodolab) [reply developers/level3.md]: oh yes, correct

File: badges/badges.md (hunk 2 above)
Comment (dodolab) [reply badges/badges.md]: sure. I'm gonna discard it and put it in another branch later on

File: developers/level3.md (hunk 4 above)
Comment (dodolab) [reply developers/level3.md:2]: Let's ask more people

File: developers/level2.md (hunk 5 above)
Comment (dodolab) [reply developers/level2.md]: I think that Dive into makes more sense:

Lean into = to push into or press against someone or something
Dive into = to occupy oneself suddenly and enthusiastically with something

File: developers/level3.md (hunk 4 above)
Comment (gilgoldman) [reply developers/level3.md:2]: Sentence structure is a bit blurry as a whole, but in this specific instance the word "as" makes more sense

File: developers/level2.md (hunk 5 above)
Comment (gilgoldman) [reply developers/level2.md]: Lean into makes more sense in this context:   

"Dive" implies we want them to drop everything at the sight of a challenge, while "Lean" suggests that if you see an opportunity to do good you should support it

**Comment by gilgoldman** [comment]
Reviewed - I think Adam is right

File: developers/level2.md (hunk 5 above)
Comment (dodolab) [reply developers/level2.md]: fixed

**Pull request merged**